### PR TITLE
Fix TimeBasedPartitioner class name in docs

### PR DESCRIPTION
**Pull request by ipostanogov**
It is case sensitive

### CODE REVIEW

**Comment by ewencp** [comment]
ok to test

@ipostanogov You can sign the CLA [here](http://clabot.confluent.io/cla).

File: docs/configuration_options.rst [hunk 1]
```diff
@@ -178,7 +178,7 @@ Connector
   * Importance: medium
 
 ``partitioner.class``
-  The partitioner to use when writing data to HDFS. You can use ``DefaultPartitioner``, which preserves the Kafka partitions; ``FieldPartitioner``, which partitions the data to different directories according to the value of the partitioning field specified in ``partition.field.name``; ``TimebasedPartitioner``, which partitions data according to the time ingested to HDFS.
+  The partitioner to use when writing data to HDFS. You can use ``DefaultPartitioner``, which preserves the Kafka partitions; ``FieldPartitioner``, which partitions the data to different directories according to the value of the partitioning field specified in ``partition.field.name``; ``TimeBasedPartitioner``, which partitions data according to the time ingested to HDFS.
```
Comment (ewencp) [inline docs/configuration_options.rst]: This file is actually generated from the config class, so there are probably corresponding changes that should be made in `HdfsSinkConnectorConfig` (or the next time someone runs the config class's main method to generate this, they may revert these changes).

**Comment by cricket007** [comment]
This can be closed.
